src/GregClient/GregClient.csproj

```diff
@@ -18,7 +18,7 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>..\..\..\Dynamo\extern\greg\</OutputPath>
+    <OutputPath>..\..\bin\Debug\</OutputPath>
```

Comment:
What about changing `OutputhPath` for Release as well? Should it not be `..\..\bin\Release`?